pyu2f/hidtransport.py

```diff
@@ -102,7 +102,7 @@ def ToWireFormat(self):
       ret[4] = self.cmd
       struct.pack_into('>H', ret, 5, self.size)
       ret[7:7 + len(self.payload)] = self.payload
-      return map(int, ret)
+      return list(map(int, ret))
```

Comment:
just curious, why is this necessary in python3?

Reply:
It's because map() returns a map object in python3, but it returns a list in python2.